cmd/kcp-catalog/bind/catalogentry/cmd.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2022 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package catalogentry
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+)
+
+var (
+	bindExampleUses = `
+	# binds to the mentioned catalog entry in the command, e.g the below command will create
+ 	# APIBindings referenced in catalog entry "certificates" present in "root:catalog:cert-manager" workspace.
+ 	%[1]s bind catalogentry root:catalog:cert-manager:certificates
```

Comment:
How would you feel about this?
```
# Binds to the 'certificates' catalog entry from the default catalog (root:catalog)
%[1]s bind catalogentry certificates

# Binds to the 'certificates' catalog entry from a specific catalog
%[1]s bind catalogentry --catalog root:some:other:catalog certificates
```

Doesn't have to be in this PR necessarily, but I like separating the catalog workspace name from the catalogentry name.

Reply:
We had some discussion around this in the design doc (https://docs.google.com/document/d/1J31wXY1-2aCyyGFjUlusKoHDzV-zktAmRdIMjMf4OR0/edit?disco=AAAAf-EcWec), and thought that since we do `roots:ws:obj` references for workspaces, we could do the same here. Based on this `kubectl-kcp bind apiexport` cmd for creating apibinding is also structured. I am fine with having it the other way round, but if we decide so we should change for the
`bind apiexport` commands too, so as to be consistent.

Reply:
Ok, we can leave it for now